docs/src/lib/presenters/Shadows.jsx

```diff
@@ -3,7 +3,8 @@ import React from 'react';
 
 export default function ( { token } ) {
 	return (
-		<div className="shadows" style={{ boxShadow: token.value }}>
+		<div>
+			<div className="shadows" style={{ boxShadow: token.value }} />
 			<pre>{ token.value }</pre>
```

Comment:
```suggestion
```

With 7fe1a2ffe0d9c17d903f931148398c7a597c2ca4 the token value is always shown, and no longer part of the presenter.

Reply:
oh cool, thanks!